### ISSUE TITLE
Gazer 0.2.19 - Dashboard import fails with 2 arguments

### ISSUE THREAD

**Issue by jakebanks** [looker-open-source/gzr#49]
Hi,

Firstly, love the tool, so thanks for that!

Anyway, with Gazer 0.2.19, `gzr dashboard help` states this for `import`:
`gzr dashboard import FILE DEST_SPACE_ID  # Import a dashboard from a file`

However, running this:
`gzr dashboard import "MyDashboard.json" 58 --host example.com --port 443 --debug --force --no-verify-ssl`

Is throwing:
```
C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/gazer-0.2.19/lib/gzr/modules/session.rb:40:in `say_warning': wrong number of arguments (given 2, expected 1) (ArgumentError)
        from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/gazer-0.2.19/lib/gzr/commands/dashboard/import.rb:49:in `execute'
        from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/gazer-0.2.19/lib/gzr/commands/dashboard.rb:60:in `import'
        from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
        from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
        from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
        from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor/invocation.rb:115:in `invoke'
        from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor.rb:238:in `block in subcommand'
        from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor/command.rb:27:in `run'
        from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor/invocation.rb:126:in `invoke_command'
        from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor.rb:387:in `dispatch'
        from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/thor-0.20.3/lib/thor/base.rb:466:in `start'
        from C:/Ruby26-x64/lib/ruby/gems/2.6.0/gems/gazer-0.2.19/exe/gzr:36:in `<top (required)>'
        from C:/Ruby26-x64/bin/gzr:23:in `load'
        from C:/Ruby26-x64/bin/gzr:23:in `<main>'
```

Thanks again

**Comment by deangelo-llooker** [comment]
Fixed by https://github.com/looker-open-source/gzr/pull/48. Thanks @ebear. Released as v0.2.20.

**Comment by jakebanks** [comment]
Thanks @deangelo-llooker @ebear
